src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices
+{
+    public static class MarshalTests
+    {
```

Comment:
Thx. I always try to reconcile inconsistency when I see it if I'm adding/modifying more stuff to an existing project.